resources/linux/rpm/code.spec.template

```diff
@@ -9,6 +9,7 @@ License:  MIT
 URL:      https://code.visualstudio.com/
 Icon:     @@NAME@@.xpm
 Requires: glibc >= 2.15
+Requires: GConf2
```

Comment:
Yes, because I'm mentioning a package name and not a library name.
When you mention a package name, if you omit the arch, the current arch will be used.
(is also possible to force the arch on a package dep but I'm not doing it on purpose).